src/index.html

```diff
@@ -7,8 +7,31 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:regular,500"
+      rel="stylesheet"
+      >
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+<header class="header">
+      <a class="logo"href="#homepage"><img src="images/logo.png"  alt="logo"></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav_item"><a href="#apple" class="is-active">Apple</a></li>
```

Comment:
You need to fix code style for all `li` items
```suggestion
          <li class="nav_item">
            <a href="#apple" class="is-active">
              Apple
            </a>
          </li>
```

Reply:
please check if it is ok now